src/app/games/diaballik/tests/diaballik.component.spec.ts

```diff
@@ -24,12 +24,10 @@ describe('DiaballikComponent', () => {
     it('should create', () => {
         testUtils.expectToBeCreated();
     });
-    it('should finish the move when clicking on the done button', fakeAsync(async() => {
+    it('should finish the move when clicking on the done button after one sub move', fakeAsync(async() => {
         // Given a state
-        // When doing some translation and then clicking on 'done'
-        console.log('click 1')
+        // When doing some submove and then clicking on 'done'
```

Comment:
Then given a state with a subMove already done
When clicking on the done button

Reply:
yep